docs/gl_objects/job_token_scope.rst

```diff
@@ -49,3 +49,24 @@ Refresh the current state of job token scope::
     scope.refresh()
     print(scope.inbound_enabled)
     # False
+
+Get a projec's CI/CD job token inbound allowlist::
+
+    allowlist = scope.allowlist.list()
```

Comment:
Not sure yet about this naming here, but it follows both the endpoint URL and our convention :confused:

Reply:
yeah it would definetly be clearer if it were `scope.projects_allowlist` since gitlab have added `groups_allowlist` https://docs.gitlab.com/ee/api/project_job_token_scopes.html#get-a-projects-cicd-job-token-allowlist-of-groups

But `scope.allowlist` does absolutely match the API design doc so it is probably best overall to be consistent with their docs 🤷